class.jetpack-signature.php

```diff
@@ -71,7 +71,17 @@ function sign_current_request( $override = array() ) {
 					$body = $_POST;
 				}
 			}
+		} else if ( 'PUT' == strtoupper( $_SERVER['REQUEST_METHOD'] ) ) {
+			// This is a little strange-looking, but there doesn't seem to be another way to get the PUT body
+			$raw_put_data = file_get_contents( 'php://input' );
```

Comment:
Good find. While this function would also be run on incoming `PUT` requests, I don't think it would cause any issues since the execution would only run during the (probably very short-lived) REST API request handler.